src/pages/costModels/costModel/updateMarkupDialog.tsx

```diff
@@ -157,8 +162,8 @@ class UpdateMarkupModelBase extends React.Component<Props, State> {
                         value="false" // "+"
                         onChange={this.handleSignChange}
                       />
-                    </div>
-                    <div style={styles.radioAlign}>
+                    </FlexItem>
+                    <FlexItem style={styles.marginTopSix}>
```

Comment:
Please rename `marginTopSix` to something more like `radioMargin`, so we know what it's used for. If we want to change the size of the margin, then we don't need to rename the property.